templates/includes/_login.html.twig

```diff
@@ -0,0 +1,29 @@
+<a class="nav-link" data-bs-toggle="modal" data-bs-target="#loginModal" href="#">
+    <i class="fas fa-user"></i>
+    Connexion</a>
+
+<!-- Modal -->
+<div class="modal fade" id="loginModal" tabindex="-1" aria-labelledby="loginModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="exampleModalLabel">Se connecter</h5>
```

Comment:
honest question : is this h5 semantic? are there h2, h3 and h4 before?

Reply:
Yes